src/app/(auth)/forgot-password/page.tsx

```diff
@@ -3,69 +3,68 @@
 import { useState } from 'react';
 import isEmail from 'validator/lib/isEmail';
 import supabase from '@/api/supabase/createClient';
-import { HorizontalDiv, SpacerDiv } from '@/app/(auth)/styles';
+import { HorizontalDiv } from '@/app/(auth)/styles';
 import { BigBlueButton, Button } from '@/components/Buttons';
 import TextInput from '@/components/TextInput/index';
 import COLORS from '@/styles/colors';
+import { Flex, SmallCardForm } from '@/styles/containers';
 import { H1, H2, H4, P } from '@/styles/text';
 
 export default function ForgotPassword() {
   const [email, setEmail] = useState('');
   const [emailSentCount, setEmailSentCount] = useState(0);
   const [emailError, setEmailError] = useState('');
 
-  const sendPasswordResetLink = async () => {
+  const handleForgotPassword = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
     if (!isEmail(email)) {
-      setEmailError('Could not find email');
+      setEmailError('Invalid email.');
```

Comment:
is there a reason why we switched to this? it looks like the design has "Could not find email"
![image](https://github.com/calblueprint/immigration-justice-project/assets/67077248/dd1913fd-95d9-4013-82cb-d80e71423dff)

Reply:
Good catch! I think "Invalid email" is only used on the Sign Up page. Can we change this back to "Could not find email"

Reply:
fixed!